README.md

```diff
@@ -55,19 +55,19 @@ ARGUMENTS for easy-stage-dataset
 ARGUMENTS for easy-stage-fileItem
 ---------------------------------
 
-    -i, --dataset-id  <arg>        id of the dataset in Fedora that should receive
-                                   the file to stage (requires file-path). If
-                                   omitted the trailing argument csf-file is
-                                   required
-        --format  <arg>            dcterms property format, the mime type of the
-                                   file
-    -p, --path-in-dataset  <arg>   the path that the file or folder should get in
-                                   the dataset
-        --path-in-storage  <arg>   Path of the file in storage, relative to the
-                                   storage-base-url, if omitted a folder is staged
-        --help                     Show help message
-        --version                  Show version of this program
-   
+    -i, --dataset-id  <arg>            id of the dataset in Fedora that should
+                                       receive the file to stage (requires
+                                       file-path). If omitted the trailing argument
+                                       csf-file is required
+    -d, --datastream-location  <arg>   http URL to redirect to
```

Comment:
So, any suggestions about the best point to check the URL scheme. You say "it's a bit late" but you are not telling where the check currently takes place.